python/glow/gwas/log_reg.py

```diff
@@ -74,10 +75,10 @@ def logistic_regression(genotype_df: DataFrame,
         - ``effect``: The effect size (if approximate Firth correction was applied)
```

Comment:
This is unrelated to this PR but occurred to me while working on WGR. If @henrydavidge and you agree, I think it is better we change the `fit_intercept` argument to `add_intercept`. I know that scikit calls it `fit_intercept` but I think it is a bit misleading because the cov matrix can have the intercept already in it and therefore fitting is actually happening in that case anyway. This should be about whether the user want us to add that column to covariates or not.